doc/ref_model/chapters/chapter04.md

```diff
@@ -282,43 +282,75 @@ The idea of the Cloud Infrastructure profiles is to have a predefined set of inf
+
 Compute Flavours can also specify secondary ephemeral storage, swap disk, etc. A compute Flavour geometry consists of the following elements:
 
 <a name="Table4-12"></a>
 
-| Element                               | Description                                                                                                                                                                                             |
+| Element                               | Description  |
 |---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Compute Flavour Name                  | A descriptive name                                                                                                                                                                                      |
-| Number of vCPUs | Number of virtual compute resources (vCPUs) presented to the VM instance.                                                                                                                               |
-| Memory                                | Virtual compute instance memory in megabytes.                                                                                                                                                           |
+| Compute Flavour Name                  | A descriptive name |
+| Compute Flavour ID                  | Flavor unique identifier |
+| Number of vCPUs | Number of virtual compute resources (vCPUs) presented to the VM instance.  |
+| Memory                                | Virtual compute instance memory in megabytes.  |
 | Ephemeral/Local Disk                  | Specifies the size of an ephemeral data disk that exists only for the life of the instance. Default value is 0.<br />The ephemeral disk may be partitioned into boot (base image) and swap space disks. |
-| Management Interface                  | Specifies the bandwidth of management interface/s                                                                                                                                                       |
-
+| Root Disk                  | Specifies the size of '/' root partition. It can be considered an Ephemeral Disk |
+| CPU Parameters                  | Capability to associate virtual CPUs to the physical CPUs, and enable specific CPU architectures |
+| NUMA                  | Allow system memory allocation into cells or nodes that are associated with particular CPUs (there could be performance degradation due to an high number of requests through this shared bus) |
+| Network Parameters                  | Enable network capabilities allowing improvements on network performance (make sure that current deployment allows it, otherwise, it can lead either to a decrease on performance or instability) |
```

Comment:
what does "allowing improvements on network performance" mean in the context of a Compute Flavour as Compute Flavours are a resource requesting mechanism.

Reply:
@pgoyal01 I've the following attributes to network parameters: 

network_DPDK; network_SRIOV; network_GPU; network_SmartNIC; network_FPGA; network_IPSec_Accel; network_Crypto_Accel; network_Transcode_Accel

Reply:
@joaoarodrigues These could be in "extra specs". I think once we have the PR #2384 merged, it would be easy to add these capabilities

Reply:
@pgoyal01 We should have these, as granular as possible, the parameters (may vary) can be available in RA1 and RA2 (specific to each of the RAn)

Reply:
@joaoarodrigues As a matter of practice only the person opening the conversation or the WSL "resolve Conversations". Thanks